source.rb

```diff
@@ -0,0 +1,14 @@
+class Source
+  attr_accessor :id, :name, :items
```

Comment:
### [Optional] Good job! here is an optional change so please make sure that the id is not private in UML diagram if it is, you should use `attr_reader ` for the id.

Reply:
Thanks, I will update it in my next commit